config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
+
+en:
+  devise:
+    confirmations:
+      confirmed: "Your email address has been successfully confirmed."
+      send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
```

Comment:
I agree, but I think this was pulled directly from `devise` gem, right @lestairon ?. It could be a good idea to bring that issue over there, maybe we can get some contributions :D